operations/rack_resiliency/Enabling_Rack_Resiliency.md

```diff
@@ -5,9 +5,9 @@ Enabling Rack Resiliency is the first step for setting up Rack Resiliency.
 **NOTE:**
 
 * By default, the Rack Resiliency feature is disabled.
-* Rack Resiliency can be enabled only during fresh install of CSM 1.7 or an
-  upgrade from CSM 1.6 to CSM 1.7.
-* Rack Resiliency cannot be disabled after it has been enabled during the install or upgrade.
+* Rack Resiliency can be enabled during fresh install of CSM 1.7 or an
+  upgrade from CSM 1.6 to CSM 1.7. Also, it can be enabled anytime after an install or upgrade.
```

Comment:
```suggestion
  upgrade from CSM 1.6 to CSM 1.7. It can also be enabled as an operational procedure on a
  system running CSM 1.7+, outside of an install or upgrade.
```